Python-Home-Challenges/app.py

```diff
@@ -0,0 +1,11 @@
+with open("./Python-Home-Challenges/Challenge_01.txt") as file:
+    fileArr = file.read().split()
+    words = {}
+    for word in fileArr:
+        if word in words:
+            words[word] += 1
+        else:
+            words[word] = 1
+    print(
+        f'''Most recurring word in that file: {max(words, key=words.get)}
+And it appears {words[max(words, key=words.get)]} times''')
```

Comment:
Fix indention here

Reply:
fixed the indentation by putting 2 different prints commands.
validated lint again